### ISSUE TITLE
Broken `latest` release (2.2.5)

### ISSUE THREAD

**Issue by LekoArts** [chakra-ui/chakra-ui#6376]
### Description

When you currently run `npm install @chakra-ui/react` it installs 2.2.5 which is broken: https://unpkg.com/browse/@chakra-ui/react@2.2.5/

It seems like this affects all packages, I also see this on e.g. `@chakra-ui/theme-tools`

### Link to Reproduction

https://unpkg.com/browse/@chakra-ui/react@2.2.5/

### Steps to reproduce

1. Install current `@chakra-ui/react` or any other Chakra UI package
2. See that 2.2.5 is installed in `node_modules` for `@chakra-ui/react` or 2.0.6 for `@chakra-ui/theme-tools`
3. That version is empty

### Chakra UI Version

All packages

### Operating System

- [x] macOS
- [ ] Windows
- [ ] Linux

```shell
  System:
    OS: macOS 12.4
    CPU: (10) arm64 Apple M1 Pro
    Shell: 5.9 - /opt/local/bin/zsh
  Binaries:
    Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
    Yarn: 3.1.1 - ~/.nvm/versions/node/v16.13.0/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.13.0/bin/npm
  Languages:
    Python: 2.7.18 - /opt/local/bin/python
  npmGlobalPackages:
    gatsby-cli: 4.17.1
    gatsby-dev-cli: 4.17.0
```

### Additional Information

I'm using yarn berry / yarn 3

**Comment by carlreid** [comment]
I believe this is the same issue as reported in issue https://github.com/chakra-ui/chakra-ui/issues/6372

**Comment by Pagebakers** [comment]
> I believe this is the same issue as reported in issue #6372

Same issue indeed, using the resolutions https://github.com/chakra-ui/chakra-ui/issues/6372#issuecomment-1195010546 as a work around for now.

**Comment by segunadebayo** [comment]
Thanks for reporting this issue. I've fixed it in the latest release (2.2.6).

Please update to the most recent version to resolve the problem. If the issue persists, I'll re-open this issue.

Cheers
